lint error

```
% mypy src
src/feditest/cli/commands/convert_transcript.py:51: error: Incompatible types in assignment (expression has type "MultifileRunTranscriptSerializer", variable has type "TestRunTranscriptSerializer | None")  [assignment]
```

--- a/src/feditest/cli/commands/convert_transcript.py
+++ b/src/feditest/cli/commands/convert_transcript.py
@@ -48,9 +48,9 @@ def run(parser: ArgumentParser, args: Namespace, remaining: list[str]) -> int:
         template = args.template
         if template == SINGLE_FILE_DEFAULT_TEMPLATE:
             template = "multifile,."
-        serializer = MultifileRunTranscriptSerializer(args.multifile, template)
+        multifile_serializer = MultifileRunTranscriptSerializer(args.multifile, template)
         try:
-            serializer.write(transcript)
+            multifile_serializer.write(transcript)
         except Exception:
             import traceback
             traceback.print_exc()

